pandera/api/dataframe/model.py

```diff
@@ -20,6 +20,8 @@
     cast,
 )
 
+import pandas as pd
```

Comment:
the generic `dataframe.model` module should not depend on any dataframe library, can you move this to https://github.com/unionai-oss/pandera/blob/main/pandera/api/pandas/model.py?

Reply:
Sure. I have done it with 38a5a95.